sources/packages/backend/nest-cli.json

```diff
@@ -88,6 +88,15 @@
       "compilerOptions": {
         "tsConfigPath": "libs/test-utils/tsconfig.lib.json"
       }
+    },
+    "integration": {
```

Comment:
"integration" is very generic. Wouldn't "fileIntegration" be better?

Reply:
mm...this was what we discussed in gromming 
![image](https://user-images.githubusercontent.com/77353155/203658200-19cb4458-846a-4cc9-be66-ac81dc7389ec.png)
. devs thoughts?

Reply:
Right now most of the integrations are based on SSH file uploads but they do not need to be.
I would say that any integration that is not a simple API call, like ATBC, can be created under this lib and that was the reason to try to keep this generic as `@sims/integration`. It is a valid point but I would keep it as stated in the ticket.